back/app/services/user_role_service.rb

```diff
@@ -1,14 +1,22 @@
 class UserRoleService
   def can_moderate?(object, user)
+    return true if user.admin?
+
     case object.class.name
     when 'Idea'
       can_moderate? object.project, user
     when 'Initiative'
       can_moderate_initiatives? user
     when 'Comment'
       can_moderate? object.post, user
+    when 'OfficialFeedback'
```

Comment:
How do you mean? They're covered here too by calling super.

Reply:
I haven't found these cases in super either.

Reply:
By calling super, the code of the user role service from the code (so the one here) is evaluated, so that's how these  cases are also being covered from there.